src/pykeen/metrics/ranking.py

```diff
@@ -307,6 +307,13 @@ def expected_value(
         n = np.mean(np.asanyarray(num_candidates)).item()
         return np.log(n) / (n - 1)
 
+    def variance(
+        self,
+        num_candidates: np.ndarray,
+        num_samples: Optional[int] = None,
+    ) -> float:
+        raise NotImplementedError
```

Comment:
Z mixins should have mean expected value zero and variance one by definition, right?

Reply:
yes they should!